src/main/webapp/app/iris/base-chatbot/iris-base-chatbot.component.ts

```diff
@@ -70,6 +75,7 @@ import * as _ from 'lodash-es';
             ]),
         ]),
     ],
+    imports: [ArtemisSharedComponentModule, ChatStatusBarComponent, IrisLogoComponent, ArtemisMarkdownModule, ArtemisSharedModule],
```

Comment:
same here with the ShareModule

Reply:
Same as above. ArtemisSharedModule is needed for artemisTranslate